msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/MessageForumsMessageManagerImpl.java

```diff
@@ -2091,4 +2092,28 @@ public List findMovedHistoryByMessageId(final Long messageid){
 
 	}
 
+	@Override
+	public void saveDraftRecipients(long msgId, List<DraftRecipient> recipients) {
+		for (DraftRecipient dr : recipients) {
+			getHibernateTemplate().persist(dr);
+		}
+	}
+
+	@Override
+	public List<DraftRecipient> findDraftRecipientsByMessageId(long msgId) {
+		HibernateCallback<List> hcb = session -> {
```

Comment:
minor this can be further shortened removing intermediate vars:
return getHibernateTemplate().execute(session -> session.getNamedQuery("findDraftRecipientsByMessageId")
           .setParameter()
          .list();